Interop/ModelCard/src/test/java/NativeModelsTest.java

```diff
@@ -0,0 +1,162 @@
+import org.junit.jupiter.api.Test;
+import org.tribuo.Dataset;
+import org.tribuo.Model;
+import org.tribuo.MutableDataset;
+import org.tribuo.Trainer;
+import org.tribuo.anomaly.evaluation.AnomalyEvaluator;
+import org.tribuo.anomaly.example.GaussianAnomalyDataSource;
+import org.tribuo.anomaly.libsvm.LibSVMAnomalyTrainer;
+import org.tribuo.anomaly.libsvm.SVMAnomalyType;
+import org.tribuo.classification.Label;
+import org.tribuo.classification.LabelFactory;
+import org.tribuo.classification.evaluation.LabelEvaluation;
+import org.tribuo.classification.evaluation.LabelEvaluator;
+import org.tribuo.classification.sgd.linear.LogisticRegressionTrainer;
+import org.tribuo.clustering.evaluation.ClusteringEvaluator;
+import org.tribuo.clustering.example.GaussianClusterDataSource;
+import org.tribuo.clustering.kmeans.KMeansTrainer;
+import org.tribuo.common.libsvm.KernelType;
+import org.tribuo.common.libsvm.SVMParameters;
+import org.tribuo.data.csv.CSVLoader;
+import org.tribuo.datasource.LibSVMDataSource;
+import org.tribuo.evaluation.TrainTestSplitter;
+import org.tribuo.math.optimisers.AdaGrad;
+import org.tribuo.math.optimisers.SGD;
+import org.tribuo.multilabel.MultiLabelFactory;
+import org.tribuo.multilabel.evaluation.MultiLabelEvaluator;
+import org.tribuo.multilabel.sgd.objectives.BinaryCrossEntropy;
+import org.tribuo.regression.RegressionFactory;
+import org.tribuo.regression.Regressor;
+import org.tribuo.regression.evaluation.RegressionEvaluator;
+import org.tribuo.regression.sgd.linear.LinearSGDTrainer;
+import org.tribuo.regression.sgd.objectives.SquaredLoss;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+
+public class NativeModelsTest {
+    @Test
+    public void testClassificationModelCard() throws IOException {
+        var factory = new LabelFactory();
+        var csvLoader = new CSVLoader<>(factory);
+
+        var irisHeaders = new String[]{"sepalLength", "sepalWidth", "petalLength", "petalWidth", "species"};
+        var irisSource = csvLoader.loadDataSource(Paths.get("src/test/input-data/bezdekIris.data"),"species", irisHeaders);
```

Comment:
We can't ship irises either, so use some generated data, or randomly generate a csv file and use that.

Reply:
I've now created a my own mock data as suggested.